py/tracks/latest.toml

```diff
@@ -27,7 +27,7 @@ full = [
     "../specs/delete_single.py",
     "../specs/delete_bulk.py",
     "../specs/any.py",
-    "../specs/count.toml",
+    "../specs/select/count.toml",
```

Comment:
just personal preference: move it down to the other specs that are in the same folder, so they are "grouped"